parking_permits/tests/test_views.py

```diff
@@ -110,7 +121,296 @@ def test_payment_view_should_update_order_and_permits_status(self):
         self.assertEqual(permit_2.status, ParkingPermitStatus.VALID)
 
 
-class RightOfPurchaseViewTestCase(APITestCase):
+class ResolvePriceViewTestCase(APITestCase):
+    def test_resolve_price_view_should_return_bad_request_if_subscription_id_missing(
+        self,
+    ):
+        url = reverse("parking_permits:talpa-price")
+        data = {
+            "orderId": "d86ca61d-97e9-410a-a1e3-4894873b1b35",
+            "userId": "d86ca61d-97e9-410a-a1e3-4894873b1b46",
+        }
+        response = self.client.post(url, data)
+        self.assertEqual(response.status_code, 400)
+
+    def test_resolve_price_view_should_return_bad_request_if_user_id_missing(self):
+        url = reverse("parking_permits:talpa-price")
+        data = {
+            "subscription_id": "f769b803-0bd0-489d-aa81-b35af391f391",
+            "orderId": "d86ca61d-97e9-410a-a1e3-4894873b1b35",
+        }
+        response = self.client.post(url, data, format="json")
+        self.assertEqual(response.status_code, 400)
+
+    def test_resolve_price_view_should_return_bad_request_if_order_meta_missing(
+        self,
+    ):
+        url = reverse("parking_permits:talpa-price")
+        data = {
+            "permitId": "80000001",
+        }
+        response = self.client.post(url, data, format="json")
+        self.assertEqual(response.status_code, 400)
+
+    def test_resolve_price_view_should_return_bad_request_if_permit_id_missing(
+        self,
+    ):
+        url = reverse("parking_permits:talpa-price")
+        data = {
+            "orderId": "d86ca61d-97e9-410a-a1e3-4894873b1b35",
+        }
+        response = self.client.post(url, data, format="json")
+        self.assertEqual(response.status_code, 400)
+
+    def test_resolve_price_view_for_normal_emission_vehicle(self):
+        talpa_subscription_id = "f769b803-0bd0-489d-aa81-b35af391f391"
```

Comment:
If these values e.g. `talpa_subscription_id` are repeated across several methods, they can just be class variables.